### ISSUE TITLE
Cannot deploy to external Postgres

### ISSUE THREAD

**Issue by Archivolt1** [prisma/prisma#4086]
**Describe the bug**
I am encountering an issue I am having a little trouble pinpointing. After updating to 1.27.0 I started having the issue described below. I tried to upgrade to the 1.28.0-beta and still encountered the same issue. I tried to roll back to 1.26.4 and I now have the issue below on that version.

**Docker File**
```
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.28-beta
    restart: always
    ports:
      - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        databases:
          default:
            connector: postgres
            host: removed
            port: 5432
            database: removed
            schema: public
            user: removed
            password: removed
            migrations: true
```

**prisma.yml**
```
endpoint: http://localhost:4466
datamodel:
  - schemas/system.prisma
  - schemas/user.prisma
secret: removed
hooks:
  post-deploy:
    - prisma generate
generate:
  - generator: typescript-client
    output: ../src/generated/prisma-client/
```

**Console Log of Docker Container**
```
Recreating removed ... 
Attaching to removed
prisma_1  | No log level set, defaulting to INFO.
prisma_1  | [INFO] Obtaining exclusive agent lock...
prisma_1  | [INFO] Initializing workers...
prisma_1  | [INFO] Successfully started 1 workers.
prisma_1  | Server running on :4466
prisma_1  | [INFO] Obtaining exclusive agent lock... Successful.
prisma_1  | [INFO] Deployment worker initialization complete.
prisma_1  | [Warning] Management authentication is disabled. Enable it in your Prisma config to secure your server.
prisma_1  | [Debug] Initializing deployment worker for default$default
prisma_1  | [Debug] Scheduling deployment for project default$default
prisma_1  | Encountered exception while applying migration. Rolling back. org.postgresql.util.PSQLException: ERROR: relation "public.ReleaseNote" does not exist
prisma_1  | org.postgresql.util.PSQLException: ERROR: relation "public.ReleaseNote" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2178)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
prisma_1  | 	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
prisma_1  | 	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
prisma_1  | 	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:155)
prisma_1  | 	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:144)
prisma_1  | 	at slick.jdbc.StatementInvoker.results(StatementInvoker.scala:38)
prisma_1  | 	at slick.jdbc.StatementInvoker.iteratorTo(StatementInvoker.scala:21)
prisma_1  | 	at slick.jdbc.Invoker.first(Invoker.scala:30)
prisma_1  | 	at slick.jdbc.Invoker.first$(Invoker.scala:29)
prisma_1  | 	at slick.jdbc.StatementInvoker.first(StatementInvoker.scala:15)
prisma_1  | 	at slick.jdbc.StreamingInvokerAction$HeadAction.run(StreamingInvokerAction.scala:52)
prisma_1  | 	at slick.jdbc.StreamingInvokerAction$HeadAction.run(StreamingInvokerAction.scala:51)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.$anonfun$run$3(DBIOAction.scala:239)
prisma_1  | 	at scala.collection.Iterator.foreach(Iterator.scala:937)
prisma_1  | 	at scala.collection.Iterator.foreach$(Iterator.scala:937)
prisma_1  | 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1425)
prisma_1  | 	at scala.collection.IterableLike.foreach(IterableLike.scala:70)
prisma_1  | 	at scala.collection.IterableLike.foreach$(IterableLike.scala:69)
prisma_1  | 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:239)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:237)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.liftedTree1$1(BasicBackend.scala:275)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:275)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | [Debug] Applied migration for project default$default
prisma_1  | [Debug] Scheduling deployment for project default$default
prisma_1  | Encountered exception while applying migration. Rolling back. org.postgresql.util.PSQLException: ERROR: relation "public.ReleaseNote" does not exist
prisma_1  | org.postgresql.util.PSQLException: ERROR: relation "public.ReleaseNote" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2178)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
prisma_1  | 	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
prisma_1  | 	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
prisma_1  | 	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:155)
prisma_1  | 	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:144)
prisma_1  | 	at slick.jdbc.StatementInvoker.results(StatementInvoker.scala:38)
prisma_1  | 	at slick.jdbc.StatementInvoker.iteratorTo(StatementInvoker.scala:21)
prisma_1  | 	at slick.jdbc.Invoker.first(Invoker.scala:30)
prisma_1  | 	at slick.jdbc.Invoker.first$(Invoker.scala:29)
prisma_1  | 	at slick.jdbc.StatementInvoker.first(StatementInvoker.scala:15)
prisma_1  | 	at slick.jdbc.StreamingInvokerAction$HeadAction.run(StreamingInvokerAction.scala:52)
prisma_1  | 	at slick.jdbc.StreamingInvokerAction$HeadAction.run(StreamingInvokerAction.scala:51)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.$anonfun$run$3(DBIOAction.scala:239)
prisma_1  | 	at scala.collection.Iterator.foreach(Iterator.scala:937)
prisma_1  | 	at scala.collection.Iterator.foreach$(Iterator.scala:937)
prisma_1  | 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1425)
prisma_1  | 	at scala.collection.IterableLike.foreach(IterableLike.scala:70)
prisma_1  | 	at scala.collection.IterableLike.foreach$(IterableLike.scala:69)
prisma_1  | 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:239)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:237)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.liftedTree1$1(BasicBackend.scala:275)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:275)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | [Debug] Applied migration for project default$default
```

**prisma deploy Output**
```
> "C:\Program Files (x86)\Yarn\bin\yarn.cmd" run prisma:deploy
yarn run v1.13.0
$ cd prisma && prisma deploy

Deploying service `default` to stage `default` to server `local`... 10.2s

Changes:

  ReleaseNote (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  User (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessional (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalSpecialties (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalLanguages (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalEducation (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalEmployment (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalOtherExp (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalCustomer (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserPaymentInformation (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`


Applying changes... (0/20)
Applying changes... 12.2s

post-deploy:

Running prisma generate...

Generating schema... 38ms
Saving Prisma Client (TypeScript) at C:\Users\itgfr\WebstormProjects\REMOVED\src\generated\prisma-client\

Running prisma generate... √

Your Prisma GraphQL database endpoint is live:

  HTTP:  http://localhost:4466
  WS:    ws://localhost:4466

Done in 29.50s.

Process finished with exit code 0 at 02:35:48.
Execution time: 29,832 ms.
```

**Expected behavior**
Should migrate and create necessary tables & columns in the Postgres database.

**Versions (please complete the following information):**
```
"prisma": "^1.28.0-beta",
"prisma-binding": "^2.3.3",
"prisma-client-lib": "^1.28.0-beta"
```

**Additional context**
I ran the prisma deploy command twice without changes hence the double log outputs. I manually inspected the database using a third party inspection tool and it appears the tables are not being created. Prisma is still able to insert/delete/modify rows from queries and mutations (from the GraphQL server) if the tables and columns are manually created. I've created a new empty schemas and altered the docker file configuration (schema option) and the tables/columns still will not generate.

**Comment by pantharshit00** [comment]
@Archivolt1 

Can you please try to remove the schema property from docker-compose file and try again?

**Comment by Archivolt1** [comment]
**With the schema property removed:**

**Altered Docker File**
```
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.28-beta
    restart: always
    ports:
      - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        databases:
          default:
            connector: postgres
            host: 165.227.215.47
            port: 5432
            database: example
            # schema: public
            user: example
            password: Removed
            migrations: true
```

**docker logs**
```
> "C:\Program Files\Docker\Docker\Resources\bin\docker-compose.exe" up --force-recreate
Recreating example_prisma_1 ... 
Attaching to example_prisma_1
prisma_1  | No log level set, defaulting to INFO.
prisma_1  | [INFO] Initializing workers...
prisma_1  | [INFO] Obtaining exclusive agent lock...
prisma_1  | [INFO] Successfully started 1 workers.
prisma_1  | Server running on :4466
prisma_1  | [INFO] Obtaining exclusive agent lock... Successful.
prisma_1  | [INFO] Deployment worker initialization complete.
prisma_1  | [Warning] Management authentication is disabled. Enable it in your Prisma config to secure your server.
prisma_1  | [Debug] Initializing deployment worker for default$default
prisma_1  | [Debug] Scheduling deployment for project default$default
prisma_1  | Encountered exception while applying migration. Rolling back. org.postgresql.util.PSQLException: ERROR: schema "example" does not exist
prisma_1  | org.postgresql.util.PSQLException: ERROR: schema "example" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2178)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
prisma_1  | 	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
prisma_1  | 	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
prisma_1  | 	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:155)
prisma_1  | 	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:144)
prisma_1  | 	at slick.jdbc.StatementInvoker.results(StatementInvoker.scala:38)
prisma_1  | 	at slick.jdbc.StatementInvoker.iteratorTo(StatementInvoker.scala:21)
prisma_1  | 	at slick.jdbc.Invoker.first(Invoker.scala:30)
prisma_1  | 	at slick.jdbc.Invoker.first$(Invoker.scala:29)
prisma_1  | 	at slick.jdbc.StatementInvoker.first(StatementInvoker.scala:15)
prisma_1  | 	at slick.jdbc.StreamingInvokerAction$HeadAction.run(StreamingInvokerAction.scala:52)
prisma_1  | 	at slick.jdbc.StreamingInvokerAction$HeadAction.run(StreamingInvokerAction.scala:51)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.$anonfun$run$3(DBIOAction.scala:239)
prisma_1  | 	at scala.collection.Iterator.foreach(Iterator.scala:937)
prisma_1  | 	at scala.collection.Iterator.foreach$(Iterator.scala:937)
prisma_1  | 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1425)
prisma_1  | 	at scala.collection.IterableLike.foreach(IterableLike.scala:70)
prisma_1  | 	at scala.collection.IterableLike.foreach$(IterableLike.scala:69)
prisma_1  | 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:239)
prisma_1  | 	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:237)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.liftedTree1$1(BasicBackend.scala:275)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:275)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | [Debug] Applied migration for project default$default
```

**prisma deploy Output**
```
> "C:\Program Files (x86)\Yarn\bin\yarn.cmd" run prisma:deploy
yarn run v1.13.0
$ cd prisma && prisma deploy

Deploying service `default` to stage `default` to server `local`... 11.8s

Changes:

  ReleaseNote (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  User (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessional (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalSpecialties (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalLanguages (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalEducation (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalEmployment (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalOtherExp (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserProfessionalCustomer (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  UserPaymentInformation (Type)
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`


Applying changes... (0/20)
Applying changes... 12.7s

post-deploy:

Running prisma generate...

Generating schema... 30ms
Saving Prisma Client (TypeScript) at C:\Users\itgfr\WebstormProjects\example\src\generated\prisma-client\

Running prisma generate... √

Your Prisma GraphQL database endpoint is live:

  HTTP:  http://localhost:4466
  WS:    ws://localhost:4466

Done in 32.34s.

Process finished with exit code 0 at 03:01:42.
Execution time: 32,596 ms.
```

**Additional Info**
`ERROR: schema "example" does not exist`
**example** is the same name of the database and user specified in the docker file.

**Comment by pantharshit00** [comment]
@Archivolt1 

Can you also remove the database field as well. Prisma will automatically create it for you.

**Comment by Archivolt1** [comment]
**Altered Docker File**
```
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.28-beta
    restart: always
    ports:
      - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        databases:
          default:
            connector: postgres
            host: 165.227.215.47
            port: 5432
            # database: example
            # schema: public
            user: example
            password: Removed
            migrations: true
```

**Docker Compose Log**
```
> "C:\Program Files\Docker\Docker\Resources\bin\docker-compose.exe" up --force-recreate
Recreating example_prisma_1 ... 
Attaching to example_prisma_1
prisma_1  | No log level set, defaulting to INFO.
prisma_1  | Feb 22, 2019 9:16:47 AM org.postgresql.core.v3.ConnectionFactoryImpl log
prisma_1  | WARNING: SQLException occurred while connecting to 165.227.215.47:5432
prisma_1  | org.postgresql.util.PSQLException: FATAL: database "prisma" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2566)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:131)
prisma_1  | 	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:210)
prisma_1  | 	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
prisma_1  | 	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
prisma_1  | 	at org.postgresql.Driver.makeConnection(Driver.java:452)
prisma_1  | 	at org.postgresql.Driver.connect(Driver.java:254)
prisma_1  | 	at slick.jdbc.DriverDataSource.getConnection(DriverDataSource.scala:101)
prisma_1  | 	at slick.jdbc.DataSourceJdbcDataSource.createConnection(JdbcDataSource.scala:68)
prisma_1  | 	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:453)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:46)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:249)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:248)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:274)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | 
prisma_1  | Feb 22, 2019 9:16:47 AM org.postgresql.Driver connect
prisma_1  | SEVERE: Connection error: 
prisma_1  | org.postgresql.util.PSQLException: FATAL: database "prisma" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2566)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:131)
prisma_1  | 	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:210)
prisma_1  | 	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
prisma_1  | 	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
prisma_1  | 	at org.postgresql.Driver.makeConnection(Driver.java:452)
prisma_1  | 	at org.postgresql.Driver.connect(Driver.java:254)
prisma_1  | 	at slick.jdbc.DriverDataSource.getConnection(DriverDataSource.scala:101)
prisma_1  | 	at slick.jdbc.DataSourceJdbcDataSource.createConnection(JdbcDataSource.scala:68)
prisma_1  | 	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:453)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:46)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:249)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:248)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:274)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | 
prisma_1  | Feb 22, 2019 9:16:47 AM org.postgresql.core.v3.ConnectionFactoryImpl log
prisma_1  | WARNING: SQLException occurred while connecting to 165.227.215.47:5432
prisma_1  | org.postgresql.util.PSQLException: FATAL: database "prisma" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2566)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:131)
prisma_1  | 	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:210)
prisma_1  | 	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
prisma_1  | 	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
prisma_1  | 	at org.postgresql.Driver.makeConnection(Driver.java:452)
prisma_1  | 	at org.postgresql.Driver.connect(Driver.java:254)
prisma_1  | 	at slick.jdbc.DriverDataSource.getConnection(DriverDataSource.scala:101)
prisma_1  | 	at slick.jdbc.DataSourceJdbcDataSource.createConnection(JdbcDataSource.scala:68)
prisma_1  | 	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:453)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:46)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:249)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:248)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:274)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | 
prisma_1  | Feb 22, 2019 9:16:47 AM org.postgresql.Driver connect
prisma_1  | SEVERE: Connection error: 
prisma_1  | org.postgresql.util.PSQLException: FATAL: database "prisma" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2566)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:131)
prisma_1  | 	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:210)
prisma_1  | 	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
prisma_1  | 	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
prisma_1  | 	at org.postgresql.Driver.makeConnection(Driver.java:452)
prisma_1  | 	at org.postgresql.Driver.connect(Driver.java:254)
prisma_1  | 	at slick.jdbc.DriverDataSource.getConnection(DriverDataSource.scala:101)
prisma_1  | 	at slick.jdbc.DataSourceJdbcDataSource.createConnection(JdbcDataSource.scala:68)
prisma_1  | 	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:453)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:46)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:249)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:248)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:274)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)
prisma_1  | 
prisma_1  | Exception in thread "main" org.postgresql.util.PSQLException: FATAL: database "prisma" does not exist
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2566)
prisma_1  | 	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:131)
prisma_1  | 	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:210)
prisma_1  | 	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
prisma_1  | 	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
prisma_1  | 	at org.postgresql.Driver.makeConnection(Driver.java:452)
prisma_1  | 	at org.postgresql.Driver.connect(Driver.java:254)
prisma_1  | 	at slick.jdbc.DriverDataSource.getConnection(DriverDataSource.scala:101)
prisma_1  | 	at slick.jdbc.DataSourceJdbcDataSource.createConnection(JdbcDataSource.scala:68)
prisma_1  | 	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:453)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:46)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:249)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:248)
prisma_1  | 	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:37)
prisma_1  | 	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:274)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
prisma_1  | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
prisma_1  | 	at java.lang.Thread.run(Thread.java:748)

Process finished with exit code 0 at 03:16:48.
Execution time: 5,577 ms.
```

**Comment by Archivolt1** [comment]
I believe I have found the solution to the issue. It is related to PostgreSQL and uppercase folding down to lowercase. I renamed all my types to lowercase only (so that the tables generated would only have fully lowercase names in the database) and now **prisma deploy** works as intended. I am unsure if this is a Prisma issue with unintended functionality or if this is a Postgres issue.

**Comment by eweilow** [comment]
I'm getting the same thing with `prismagraphql/prisma:1.27`, but not on `prismagraphql/prisma:1.26.4`

**Comment by pantharshit00** [comment]
Cc @do4gr, Looks like this is caused to the recent database property fix you did.

**Comment by michaelspeed** [comment]
this is happening to prisma >1.25, postgres migration is not working

**Comment by do4gr** [comment]
I'll have a look. Thanks for the detailed reports!

**Comment by do4gr** [comment]
We identified and fixed one issue. But from your description it's not 100% clear whether that was the problem you ran into. 

The combination of providing a schema name and migrations was a new feature for us. The connector was not actually creating a schema with the provided name but was assuming that there is already a schema with that name that it could just write into. We changed that so that a schema with the provided name will be created if it does not exist already. 

The second issue was that we did not account for the option to provide a database but no schema. This should now create a schema with a autogenerated name in the provided database. 

Does one of the two match your circumstances?

We'll release a fix for these two shortly.
